### PR TITLE
Use official metrics-jcache

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: fullstack/pom.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
         <awaitility.version>1.7.0</awaitility.version>
         <commons-io.version>2.5</commons-io.version>
         <commons-lang.version>3.4</commons-lang.version>
+        <ehcache3.version>3.3.0</ehcache3.version>
```
Comment (anthonydahanne) [inline fullstack/pom.xml]: latest spring not using ehcache 3.3.0 ?

File: fullstack/src/main/java/org/terracotta/demo/config/MetricsConfiguration.java [hunk 2]
```diff
@@ -8,14 +8,14 @@
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.terracotta.demo.config.jcache.JCacheGaugeSet;
```
Comment (anthonydahanne) [inline fullstack/src/main/java/org/terracotta/demo/config/MetricsConfiguration.java]: un necessary re order ;-)

Comment (henri-tremblay) [inline fullstack/src/main/java/org/terracotta/demo/config/MetricsConfiguration.java]: Yes. Necessary. It's not in the same package anymore

File: fullstack/pom.xml (hunk 1 above)
Comment (henri-tremblay) [inline fullstack/pom.xml]: No. It's 3.2.1. We should PR them probably

File: fullstack/src/main/java/org/terracotta/demo/config/MetricsConfiguration.java (hunk 2 above)
Comment (anthonydahanne) [inline fullstack/src/main/java/org/terracotta/demo/config/MetricsConfiguration.java]: ah right, my bad

File: fullstack/pom.xml (hunk 1 above)
Comment (anthonydahanne) [inline fullstack/pom.xml]: true

**Pull request merged**